src/manage.c

```diff
@@ -177,7 +178,7 @@ u8 best_prn(void)
       if (acq_prn_param[prn].state == ACQ_PRN_TRIED)
         acq_prn_param[prn].state = ACQ_PRN_UNTRIED;
     }
-    printf("acq: restarting PRN search\n");
+    log_info("acq: restarting PRN search\n");
```

Comment:
I think this is a good idea but I might follow up with those changes in a separate PR